cvc5_pythonic_api/cvc5_pythonic.py

```diff
@@ -6267,6 +6273,8 @@ def setOption(self, name=None, value=None, **kwargs):
             kwargs[name] = value
         for k, v in kwargs.items():
             _assert(isinstance(k, str), "non-string key " + str(k))
+            if k == 'unsat_core':
```

Comment:
Why is this needed? I think the standard option name is `produce-unsat-cores` and not `unsat_core`. Does `z3` allow the user to turn on an option called `unsat_core`? Where?

Reply:
Moreover, we don't actually support unsat cores right? only unsat assumptions?